app/controllers/batch_controller.rb

```diff
@@ -0,0 +1,37 @@
+class BatchController < ApplicationController
+  def create
+    uris = payload.fetch(:uris)
+    checked_within = (payload[:checked_within] || 24.hours).to_i
+    callback_uri = payload[:callback_uri]
```

Comment:
it'd be nice to validate these - rails doesn't make that super easy outside of db models. But a quick google around shows this might be pretty good: http://api.rubyonrails.org/classes/ActiveModel/Validations.html

failing that this is a less railsy alternative: http://dry-rb.org/gems/dry-validation/